src/main/scala/mesosphere/marathon/MarathonSchedulerService.scala

```diff
@@ -66,19 +53,18 @@ trait LeadershipAbdication {
 class MarathonSchedulerService @Inject() (
   leadershipCoordinator: LeadershipCoordinator,
   healthCheckManager: HealthCheckManager,
-  @Named(ModuleNames.CANDIDATE) candidate: Option[Candidate],
   config: MarathonConf,
   frameworkIdUtil: FrameworkIdUtil,
-  @Named(ModuleNames.LEADER_ATOMIC_BOOLEAN) leader: AtomicBoolean,
+  electionService: ElectionService,
+  prePostDriverCallbacks: Seq[PrePostDriverCallback],
   appRepository: AppRepository,
   driverFactory: SchedulerDriverFactory,
   system: ActorSystem,
   migration: Migration,
   @Named("schedulerActor") schedulerActor: ActorRef,
   @Named(EventModule.busName) eventStream: EventStream,
```

Comment:
The `MarathonSechedulerService` doesn't use the eventbus anymore , so we can get rid of this dependency.